sass/_legacy-docs.scss

```
@@ -1,10 +1,10 @@
 .legacy-doc-warning {
-    padding: 20px;
-    border-radius: 10px;
+    padding: 1rem;
+    border-radius: 0.5rem;
```

```suggestion
    border-radius: .5rem;
```

To maintain a consistent style?